index.js

```diff
@@ -273,6 +274,8 @@ export default {
           //    '0x0' RLP-encoded transaction of all fields in Transaction (minus version, v, r, s)
           //    '0x1' typed encoding matching MetaMask initial implementation of EIP-712
           rawTx.version = '0x1';
+          rawTx.transferTokenId="0x8bb0";
```

Comment:
also not needed because of line 251

Reply:
![image](https://user-images.githubusercontent.com/19581409/65413375-e5ac3280-de23-11e9-9385-8ad3e44627f7.png)

Reply:
but in npm , it is not used

Reply:
this is a  question, should  update npm , add "const rawTx = Object.assign({}, defaultTokenSetting, obj);"  and  "defaultTokenSetting"